pytorch_lightning/logging/mlflow_logger.py

```diff
@@ -3,8 +3,8 @@
 
 try:
     import mlflow
-except ImportError:
-    raise ImportError('Missing mlflow package.')
+except Exception as e:
```

Comment:
you do not need the `e`, PEP8 will complain that you define a variable which is not used